packages/backend/src/server/web/views/page.pug

```diff
@@ -17,7 +17,6 @@ block og
 	meta(property='og:description' content= page.summary)
 	meta(property='og:url'         content= url)
 	meta(property='og:image'       content= page.eyeCatchingImage ? page.eyeCatchingImage.thumbnailUrl : user.avatarUrl)
-
```

Comment:
Looks like this blank line shouldn't be erased.

Reply:
Not sure why it was. Added it back.